modules/light-clients/09-localhost/client_state.go

```diff
@@ -66,20 +66,22 @@ func (cs ClientState) Initialize(ctx sdk.Context, cdc codec.BinaryCodec, clientS
 	return nil
 }
 
-// GetTimestampAtHeight must return the timestamp for the consensus state associated with the provided height.
+// GetTimestampAtHeight returns the current block time retrieved from the application context. The localhost client does not store consensus states and thus
+// cannot provide a timestamp for the provided height.
 func (cs ClientState) GetTimestampAtHeight(ctx sdk.Context, clientStore sdk.KVStore, cdc codec.BinaryCodec, height exported.Height) (uint64, error) {
 	return uint64(ctx.BlockTime().UnixNano()), nil
 }
 
 // VerifyMembership is a generic proof verification method which verifies a proof of the existence of a value at a given CommitmentPath at the specified height.
 // The caller is expected to construct the full CommitmentPath from a CommitmentPrefix and a standardized path (as defined in ICS 24).
+// Note the store provided to the localhost client is the full IBC core state store.
```

Comment:
```suggestion
// VerifyMembership is a generic proof verification method which verifies the existence of a given key and value within the IBC store. The caller is expected to construct the full CommitmentPath from a CommitmentPrefix 
// and a standardized path (as defined in ICS 24). The caller must provide the full IBC store. 
```
something like this? Sorry the spacing is hard to do on github